ourgroceries/__init__.py

```diff
@@ -105,23 +105,27 @@ async def _get_team_id(self):
         async with aiohttp.ClientSession(cookies=cookies) as session:
             async with session.get(YOUR_LISTS) as resp:
                 responseText = await resp.text()
-                for team_id in re.findall(REGEX_TEAM_ID, responseText):
-                    self._team_id = team_id
-                    _LOGGER.debug('ourgroceries found team_id {}'.format(self._team_id))
+                self._team_id = re.findall(REGEX_TEAM_ID, responseText)[0]
+                self._category_id = re.findall(REGEX_CATEGORY_ID, responseText)[0]
+
 
     async def get_my_lists(self):
         """Get our grocery lists."""
         _LOGGER.debug('ourgroceries get_my_lists')
         return await self._post(ACTION_GET_LISTS)
 
+    async def get_category_items(self):
+        """Get category items."""
+        _LOGGER.debug('ourgroceries get_list_items')
```

Comment:
change `_LOGGER.debug('ourgroceries get_list_items')` to `_LOGGER.debug('ourgroceries get_category_items')`